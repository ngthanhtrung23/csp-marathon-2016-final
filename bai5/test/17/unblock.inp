.11.22
333.45
670045
67..49
6.ABB9
CCA.DD
