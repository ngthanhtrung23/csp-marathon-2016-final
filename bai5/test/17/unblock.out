25
1 L 1
2 L 1
7 D 1
A U 1
C R 1
6 D 1
0 L 2
A U 1
B L 1
4 D 1
D L 1
9 D 1
5 D 1
3 R 3
A U 2
0 R 1
6 U 1
C L 1
D L 1
4 D 1
0 R 2
A D 2
3 L 1
5 U 2
0 R 1

